### PR TITLE
feat: circle icons and dropdown adjustment

### PR DESCRIPTION
**Pull request by alfonsobries**
## Summary
Related to: https://app.clickup.com/t/qth0ru
Dependant of:  https://github.com/ArkEcosystem/nodem/pull/142

- Icons needed for server log detail statues
- New property for changing the origin class in the dropdown (used on the table filters)

## Checklist

-   [x] I checked my UI changes against the design and there are no notable differences
-   [x] I checked my UI changes for any responsiveness issues
-   [x] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I provided a screenshot of my changes to the component _(if applicable)_
-   [ ] I regenerated the `icons.html` file and checked if my newly added icon is shown correctly _(if necessary)_
-   [ ] I added an explanation on how to use the component to the readme _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

File: resources/views/dropdown.blade.php [hunk 1]
```diff
@@ -4,6 +4,7 @@
     'buttonClassExpanded'    => 'text-theme-primary-500',
     'buttonClass'            => 'text-theme-secondary-400 hover:text-theme-primary-500',
     'dropdownClasses'        => 'w-40',
+    'dropdownOriginClass'    => 'origin-top-right',
```
Comment (leMaur) [inline resources/views/dropdown.blade.php:7]: 🤔 Where is `dropdownOriginClass` used?

**Review by leMaur — CHANGES_REQUESTED** [review]

**Comment by alfonsobries** [comment]
@leMaur forget to add the link https://github.com/ArkEcosystem/nodem/pull/142 (the branch is not ready)

File: resources/views/dropdown.blade.php (hunk 1 above)
Comment (alfonsobries) [reply resources/views/dropdown.blade.php:7]: @leMaur its used here ArkEcosystem/nodem#142

File: resources/assets/icons/status/debug.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg version="1.1" id="icon" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 20 20" xml:space="preserve"><g transform="translate(1.5 1.5)" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10"><circle cx="8.5" cy="8.5" r="9"/><path d="M8.5 8.5V5.2m0 3.3l3.9 3.9"/></g></svg>
```
Comment (ItsANameToo) [inline resources/assets/icons/status/debug.svg:1]: identical to 
image

File: resources/assets/icons/status/error.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg version="1.1" id="icon" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 20 20" xml:space="preserve"><g fill="none" stroke="currentColor" stroke-width="2"><circle stroke-miterlimit="10" cx="10" cy="10" r="9"/><path stroke-linecap="round" stroke-linejoin="round" d="M7 13l6-6m0 6L7 7"/></g></svg>
```
Comment (ItsANameToo) [inline resources/assets/icons/status/error.svg:1]: identical to 
image

File: resources/assets/icons/status/fatal.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg version="1.1" id="icon" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 20 20" xml:space="preserve"><path d="M10 19c5 0 9-4 9-9s-4-9-9-9-9 4-9 9h0c0 5 4 9 9 9zm-6.4-2.6L16.4 3.6" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/></svg>
```
Comment (ItsANameToo) [inline resources/assets/icons/status/fatal.svg:1]: identical to 
image

File: resources/assets/icons/status/info.svg [hunk 5]
```diff
@@ -0,0 +1 @@
+<svg version="1.1" id="icon" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 20 20" xml:space="preserve"><g fill="none" stroke="currentColor" stroke-width="2"><circle stroke-miterlimit="10" cx="10" cy="10" r="9"/><path stroke-linecap="round" stroke-miterlimit="10" d="M10 9.2v5"/><path stroke-linecap="round" stroke-linejoin="round" d="M10 5.8h0"/></g></svg>
```
Comment (ItsANameToo) [inline resources/assets/icons/status/info.svg:1]: identical to
image

File: resources/assets/icons/status/trace.svg [hunk 6]
```diff
@@ -0,0 +1 @@
+<svg version="1.1" id="icon" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 20 20" xml:space="preserve"><circle fill="none" stroke="currentColor" stroke-width="2" stroke-miterlimit="10" cx="10" cy="10" r="9"/><path fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" d="M9.8 13.5c2.1 0 3.8-1.7 3.8-3.8 0-2.1-1.7-3.8-3.8-3.8C7.7 5.9 6 7.6 6 9.7c0 2.1 1.7 3.8 3.8 3.8zm4.2.6L12.9 13"/></svg>
```
Comment (ItsANameToo) [inline resources/assets/icons/status/trace.svg:1]: I guess this one is the only new one 🤔

File: resources/assets/icons/status/warning.svg [hunk 7]
```diff
@@ -0,0 +1 @@
+<svg version="1.1" id="icon" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 20 20" xml:space="preserve"><g fill="none" stroke="currentColor" stroke-width="2"><circle stroke-miterlimit="10" cx="10" cy="10" r="9"/><path stroke-linecap="round" stroke-miterlimit="10" d="M10 10.5v-5"/><path stroke-linecap="round" stroke-linejoin="round" d="M10 14h0"/></g></svg>
```
Comment (ItsANameToo) [inline resources/assets/icons/status/warning.svg:1]: identical to 
image

**Review by ItsANameToo — CHANGES_REQUESTED** [review]
preferably:
- do not add new icons that are duplicates
- rename the toast/notification ones to be more generic, and update the respective components to make use of those so we can keep it all consistent by making them use the same icon

File: resources/assets/icons/status/fatal.svg (hunk 4 above)
Comment (alfonsobries) [reply resources/assets/icons/status/fatal.svg:1]: @ItsANameToo this one is not equal, the blocked one has wider lines

File: resources/assets/icons/status/debug.svg (hunk 2 above)
Comment (alfonsobries) [reply resources/assets/icons/status/debug.svg:1]: @ItsANameToo its not identical, this one has wider lines
Waiting restart
image
Debug:
image

File: resources/assets/icons/status/fatal.svg (hunk 4 above)
Comment (alfonsobries) [reply resources/assets/icons/status/fatal.svg:1]: blocked:
image
Fatal:
image

Comment (ItsANameToo) [reply resources/assets/icons/status/fatal.svg:1]: then it's up to oleg to define which one to use, they look identical otherwise 🤔

**Comment by alfonsobries** [comment]
@ItsANameToo other than the two icons I commented merged the toast icons with the new ones added here inside the `status` folder and updated the references for the toasts

File: resources/assets/icons/status/debug.svg (hunk 2 above)
Comment (ItsANameToo) [reply resources/assets/icons/status/debug.svg:1]: can you ask oleg which one to use, they have identical designs

File: resources/assets/icons/status/fatal.svg (hunk 4 above)
Comment (alfonsobries) [reply resources/assets/icons/status/fatal.svg:1]: @ItsANameToo the one that I added is the only one that looks good, the one for `notifications/blocked` looks weird (same for the clock icon below)

image

File: resources/assets/icons/status/trace.svg [hunk 8]
```diff
@@ -1 +1 @@
-<svg version="1.1" id="icon" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 20 20" xml:space="preserve"><circle fill="none" stroke="currentColor" stroke-width="2" stroke-miterlimit="10" cx="10" cy="10" r="9"/><path fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" d="M7 13l6-6m0 6L7 7"/></svg>
```
Comment (alexbarnsley) [inline resources/assets/icons/status/trace.svg:1]: I'd rename this to "search"

File: resources/views/dropdown.blade.php [hunk 9]
```diff
@@ -68,7 +69,7 @@ class="flex items-center focus:outline-none dropdown-button transition-default {
         x-transition:leave="transition ease-in duration-75"
         x-transition:leave-start="transform opacity-100 scale-100"
         x-transition:leave-end="transform opacity-0 scale-95"
-        class="origin-top-right absolute right-0 mt-2 z-10 dropdown {{ $dropdownClasses }} {{ $fullScreen ? 'w-screen -mx-8 md:w-auto md:mx-0' : '' }}"
+        class=" absolute right-0 mt-2 z-10 dropdown {{ $dropdownClasses }} {{ $fullScreen ? 'w-screen -mx-8 md:w-auto md:mx-0' : '' }}"
```
Comment (alexbarnsley) [inline resources/views/dropdown.blade.php:72]: trim the space at the start

**Review by alexbarnsley — CHANGES_REQUESTED** [review]

File: resources/assets/icons/status/debug.svg (hunk 2 above)
Comment (alexbarnsley) [reply resources/assets/icons/status/debug.svg:1]: FYI I asked Oleg about this

File: resources/assets/icons/status/fatal.svg (hunk 4 above)
Comment (alexbarnsley) [reply resources/assets/icons/status/fatal.svg:1]: FYI I asked Oleg about which one to use

**Comment by alfonsobries** [comment]
@ItsANameToo as @alexbarnsley  proposed I moved the icons to a `circle` folder and renamed the icons to something more general:

- search.svg
- checkmark.svg
- clock.svg
- cross.svg
- exclamation-mark.svg
- forbidden.svg
- info.svg
- question-mark.svg


Also created a new PR on nodem to handle these new names: https://github.com/ArkEcosystem/nodem/pull/147

File: resources/views/toast.blade.php [hunk 10]
```diff
@@ -7,12 +7,12 @@
...
-        'danger' => 'toasts.danger',
-        'success' => 'toasts.success',
-        'info' => 'toasts.info',
-        'hint' => 'toasts.hint',
+        'warning' => 'status.warning',
+        'error' => 'status.error',
+        'danger' => 'status.error',
+        'success' => 'status.success',
+        'info' => 'status.info',
+        'hint' => 'status.hint',
```
[the remark this answers is not in the document]
Comment (alfonsobries) [reply resources/views/toast.blade.php]: @alexbarnsley like the idea of the circle folder

**Comment by ItsANameToo** [comment]
@alfonsobries moved the `circle/clock` icon to just `clock` as it's not a clock inside a circle

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**